src/core/ext/filters/channel_idle/channel_idle_filter.cc

```diff
@@ -173,21 +174,24 @@ void MaxAgeFilter::PostInit() {
               GRPC_CHANNEL_STACK_REF(this->channel_stack(),
                                      "max_age send_goaway");
               // Jump out of the activity to send the goaway.
-              auto fn = [](void* arg, grpc_error_handle) {
-                auto* channel_stack = static_cast<grpc_channel_stack*>(arg);
-                grpc_transport_op* op = grpc_make_transport_op(nullptr);
-                op->goaway_error = grpc_error_set_int(
-                    GRPC_ERROR_CREATE_FROM_STATIC_STRING("max_age"),
-                    GRPC_ERROR_INT_HTTP2_ERROR, GRPC_HTTP2_NO_ERROR);
-                grpc_channel_element* elem =
-                    grpc_channel_stack_element(channel_stack, 0);
-                elem->filter->start_transport_op(elem, op);
-                GRPC_CHANNEL_STACK_UNREF(channel_stack, "max_age send_goaway");
+              struct Closure {
```

Comment:
This is a poor choice of name (here and elsewhere). There's no closure here. I'll replace this with something else in a subsequent commit to this pull request. Open to suggestions.